src/models/Challenge.js

```diff
@@ -0,0 +1,12 @@
+import { Schema, model } from "mongoose";
+
+const schema = new Schema({
+  creator_id: { type: String, required: true },
+  name: { type: String, required: true },
+  description: { type: String, required: true },
+  start_at: { type: String, required: true },
+  end_at: { type: String, required: true },
```

Comment:
We should add the status, required false, and default as ongoing

enum example for the status field: https://mongoosejs.com/docs/validation.html

Reply:
updated: https://github.com/LangaraBeetles/weaup-api/pull/7/commits/d3fa89d92bd41c8962d552f2b2504e53fb7574cb